feature/search/src/main/java/com/google/samples/apps/nowinandroid/feature/search/SearchScreen.kt

```diff
@@ -128,26 +305,29 @@ private fun SearchTextField(onSearchQueryChanged: (String) -> Unit) {
             )
         },
         trailingIcon = {
-            IconButton(onClick = { textState.value = "" }) {
+            IconButton(onClick = {
+                searchQuery.value = ""
```

Comment:
Use viewModel.searchQuery as single source of truth
```suggestion
```

Reply:
Done